scripts/build_darwin.sh

```diff
@@ -8,7 +8,7 @@ usage() {
     exit 1
 }
 
-export VERSION=${VERSION:-$(git describe --tags --dirty)}
+export VERSION=${VERSION:-$(git describe --tags --first-parent --abbrev=7 --long --dirty --always | sed -e "s/^v//g")}
```

Comment:
Does this effect the release download URL? I am wondering how this effects the auto-updater

Reply:
No, this only impacts local builds when you don't set VERSION explicitly.

The bug is darwin was providing 8 characters of git hash vs. 7 for linux and windows, so the version strings weren't the same, even though it was built from the same commit.